assignment/assignment-tool/tool/src/webapp/js/assignments.js

```diff
@@ -483,3 +488,303 @@ function enableLinks()
         }
     }
 }
+
+// SAK-29314
+function navPanelAction( submissionID, action )
```

Comment:
these (and all other javascript functions) should probably be namespaced by assigning them to a var. Convention would be to use an uppercase variable name like ASN.